cardazim/server.py

```diff
@@ -0,0 +1,51 @@
+import socket
+import argparse
+import sys
+
+def listener_server(server_ip: str, server_port: int):
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as server_socket:
+        server_socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        server_socket.bind((server_ip, server_port))
+        server_socket.listen(0)
```

Comment:
why using listen(0)?

Reply:
because I don't want to support queued connections.

Reply:
I'll change it to 5.